core/templates/core/profile.html

```diff
@@ -0,0 +1,27 @@
+{% extends 'core/base.html' %}
+
+{% block title %}{{ user.name }}{% endblock %}
+
+{% block content %}
+    <div class="mt-32 w-screen flex flex-col items-center justify-center">
+        <div class="flex gap-5 w-1/2 pb-4 border-b-2">
+            <img src="{{ user.avatar }}" alt="avatar" class="h-auto w-1/8 rounded-full">
+            <div class="flex flex-col">
+                <h1 class="text-4xl text-bold">{{ user.name }}</h1>
+                <div class="flex gap-4 text-gray-500 flex-wrap">
+                    {% for role in user.roles %}
+                        <p>{{ role }}</p>
+                    {% endfor %}
+                </div>
+            </div>
+        </div>
+        <div class="flex flex-col w-1/2 gap-5">
+            <div class="flex gap-5 mt-4 text-yellow-500 justify-end">
+                <p>Level: {{ user.level }}</p>
+                <p>Experience: {{ user.exp }}</p>
+            </div>
+            <progress class="w-full" value="{{ user.exp }}" max="2137">{{ user.exp }} / 2137</progress>
```

Comment:
- Round corners
- Make background darker
- Add nice border
- Make progress color a gradient
- Maybe add nice animation from 0 to current %?

Reply:
Border doesn't work on this, I can't do animations, I don't know how to change progress color

Reply:
So don't use progress tag and implement it manualy (div inside div with fixed width)